gradle-baseline-java/src/main/groovy/com/palantir/baseline/IntellijSupport.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * (c) Copyright 2021 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.baseline;
+
+public final class IntellijSupport {
+    private static final String INTELLIJ_ACTIVE = "idea.active";
+
+    public static boolean isRunningInIntellij() {
```

Comment:
Mind consolidating the `idea.active` checks from `BaselineIdea` as well?

Reply:
of course! done